src/torchcodec/decoders/_core/VideoDecoder.cpp

```diff
@@ -1090,6 +1090,45 @@ VideoDecoder::BatchDecodedOutput VideoDecoder::getFramesAtIndices(
   return output;
 }
 
+VideoDecoder::BatchDecodedOutput VideoDecoder::getFramesDisplayedByTimestamps(
+    int streamIndex,
+    const std::vector<double>& framePtss) {
```

Comment:
I'm not super happy about this parameter name `framePtss` in C++, `frame_ptss` in Python. It's the mirror of `frame_indices`. Happy to consider alternatives. The "single frame" equivalent in `get_frame_displayed_at` is just `seconds`.

Reply:
Maybe just `timestamps` since it's in the function name too?

Reply:
It should probably be `framePtses` and `frame_ptses`, following the English rule of adding "es" to make something that ends in "s" plural. That's still not great, but the current version definitely looks like a typo.

Reply:
I'm happy with `timestamps` - there was a race condition in commenting. :)

Reply:
I think `timestamps` is a bit more intuitive to me than `frame_timestamps` because these are random time points given by the user -- not necessarily corresponding to `frame.pts` or `frame.pts + frame.duration` (they could be somewhere in between).

Heck they could even be outside the min and max pts.

But whatever you choose is okay

Reply:
SG, I'll go with `timestamps` then